includes/amp-helper-functions.php

```diff
@@ -1130,3 +1130,34 @@ function amp_add_admin_bar_view_link( $wp_admin_bar ) {
 
 	$wp_admin_bar->add_menu( $parent );
 }
+
+/**
+ * Prints AMP Stories auto ads.
+ *
+ * @since 1.2
+ */
+function amp_print_story_auto_ads() {
+	/**
+	 * Filters the configuration data for <amp-story-auto-ads>
+	 *
+	 * This allows Dynamically inserting ads into a story.
+	 *
+	 * @param array   $data Story ads configuration data.
+	 * @param WP_Post $post The current story's post object.
+	 */
+	$data = apply_filters( 'amp_stories_auto_ads_configuration', array(), get_post() );
```

Comment:
I think this should be `amp_story` instead of `amp_stories`, following example in #2468.

It's the filter for a single story since `get_post()` is passed in.